arangod/Transaction/Methods.cpp

```diff
@@ -4008,8 +4003,7 @@ Future<OperationResult> Methods::removeInternal(
   if (_state->isCoordinator()) {
     f = removeCoordinator(collectionName, value, options, api);
   } else {
-    OperationOptions optionsCopy = options;
-    f = removeLocal(collectionName, value, optionsCopy);
+    f = removeLocal(collectionName, value, options);
```

Comment:
nit: is it possible to std::move the `options` here into `removeLocal(...)`? That would save a copy of the options object, which contains strings with heap-allocations attached to them.